### DISCUSSION TITLE
Website bug: Ads make site jump up/down every 5 seconds on tablet view

### DISCUSSION THREAD

**Discussion by corneliusroemer** [fastapi/fastapi#8712] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
Not a code bug
```

### Description

Open the tutorial website on a tablet in vertical mode (e.g. iPad)

Observe that the ads at the top make the whole site jump up and down every 5 seconds or so.

This is excruciating and making it impossible for me to read the docs.

Ads should be switched off until this is fixed, it totally degrades the user experience.

### Operating System

Other

### Operating System Details

iPad OS 14

### FastAPI Version

Web

### Python Version

Web

**Comment by corneliusroemer** [comment]
![C96979A3-E447-4EF6-B892-5FB2856B49A2](https://user-images.githubusercontent.com/25161793/146863327-74c0f0e9-b614-434d-83af-dc6745bc02f3.jpeg)
![0E28CBA1-4529-463A-A4DE-E5599372ADA3](https://user-images.githubusercontent.com/25161793/146863332-f22d7d46-3e20-45ad-88d7-46aa124868fb.jpeg)
![4C1CA926-391F-4BA6-8DDC-D771E8DFA58D](https://user-images.githubusercontent.com/25161793/146863334-02afe3c9-c7e6-4bdb-87ec-dedcb7900709.jpeg)

**Comment by stevesimmons** [comment]
It is not just a tablet error. The same vertical jitter occurs in a regular browser when the window is made narrow.

There seem to be two distinct problems:
* The ad/sponsor box has two slots. side by side in a normal width window. If the window is narrow, they stack vertically in two rows. If the window has an intermediate width, this flip between one and two rows if some ads are wider than than others.
* For me at last, the ads cycle periodically through a list of several. One of them appears to be a bad link. So when it is that ad's turn to appear, the vertical height goes to zero, and the whole docs page shifts up by 1cm.

Solutions are probably (A) force all ads/messages to the same dimension; (B) fix the total height of this box. (Or (C), drop the ads altogether :) ).

**Comment by corneliusroemer** [comment]
Thanks for investigating. I agree with your findings.

I think the reason no one has opened an issue for this before me is that to open an issue you have to jump through 10 hoops including committing to help with other issues. One basically has to lie to open an issue here :/
